src/modules/client/dto/candidate-filter.dto.ts

```diff
@@ -0,0 +1,12 @@
+import { IsArray, IsString } from 'class-validator';
```

Comment:
IsArray is not used, check if your eslint is configured, it would give a warning on this situation

Reply:
Yes, it give a warning. Just didn't see